app/services/favourite_service.rb

```diff
@@ -37,11 +37,8 @@ def create_notification(favourite)
     end
   end
 
-  def bump_potential_friendship(account, status)
+  def bump_potential_friendship(_account, _status)
     ActivityTracker.increment('activity:interactions')
-    return if account.following?(status.account_id)
-
-    PotentialFriendshipTracker.record(account.id, status.account_id, :favourite)
```

Comment:
This method should probably be removed or renamed, since it doesn't bump anything but the global interaction count.